docker/docker-compose-scopes-example.yml

```diff
@@ -0,0 +1,87 @@
+version: "3.8"
+volumes:
+  opal:
+services:
+  # When scaling the opal-server to multiple nodes and/or multiple workers, we use
+  # a *broadcast* channel to sync between all the instances of opal-server.
+  # Under the hood, this channel is implemented by encode/broadcaster (see link below).
+  # At the moment, the broadcast channel can be either: postgresdb, redis or kafka.
+  # The format of the broadcaster URI string (the one we pass to opal server as `OPAL_BROADCAST_URI`) is specified here:
+  # https://github.com/encode/broadcaster#available-backends
+  broadcast_channel:
```

Comment:
As we already discussed - I think that having 2 different database backends for `opal-server` looks & smells really bad.
At least in the example: get rid of postgres, use redis for the broadcast channel as well!